### ISSUE TITLE
ValueError when download a specific album.

### ISSUE THREAD

**Issue by AutisticCoder** [Sorrow446/Qo-DL#123]
` File "Qo-DL.py", line 799, in <module>
    init()
  File "Qo-DL.py", line 792, in init
    preRip(appId, album_id, isTrack, isDiscog, isPlist, session, comment, formatId, alcovs, downloadDir, keepCover, folderTemplate, filenameTemplate, appSecret)
  File "Qo-DL.py", line 494, in preRip
    rip(album_id, isTrack, isDiscog, isPlist, session, comment, formatId, alcovs, downloadDir, keepCover, folderTemplate, filenameTemplate, "", "", appSecret)
  File "Qo-DL.py", line 408, in rip
    add_flac_tags(temporary_filename, metadata)
  File "Qo-DL.py", line 146, in add_flac_tags
    audio[tag] = value
  File "C:\Users\z5205847\AppData\Roaming\Python\Python37\site-packages\mutagen\_file.py", line 75, in __setitem__
    self.tags[key] = value
  File "C:\Users\z5205847\AppData\Roaming\Python\Python37\site-packages\mutagen\_vorbis.py", line 306, in __setitem__
    raise ValueError
ValueError`
Album I'm trying to download is 
"https://open.qobuz.com/album/5099972963753"
I have all the tags including extended metadata enabled. 
Using a studio account

**Comment by girpa3** [comment]
Hey DshLt, could you please update the Ti-DL config file to resolve the error of bad credentials.
I know this is not the place to comment here but that project is archived.

**Comment by Superhackman** [comment]
> Hey DshLt, could you please update the Ti-DL config file to resolve the error of bad credentials.
> I know this is not the place to comment here but that project is archived.

I do not think it fixes, since it is archived and when a repository is archived it is possible that the developer abandoned it and archived it for maybe in the future it will repair it, this is the developer's decision

**Comment by AutisticCoder** [comment]
> 
> 
> > Hey DshLt, could you please update the Ti-DL config file to resolve the error of bad credentials.
> > I know this is not the place to comment here but that project is archived.
> 
> I do not think it fixes, since it is archived and when a repository is archived it is possible that the developer abandoned it and archived it for maybe in the future it will repair it, this is the developer's decision

Can you autists not spam my issue with this shit.

**Comment by Superhackman** [comment]
> > > Hey DshLt, could you please update the Ti-DL config file to resolve the error of bad credentials.
> > > I know this is not the place to comment here but that project is archived.
> > 
> > 
> > I do not think it fixes, since it is archived and when a repository is archived it is possible that the developer abandoned it and archived it for maybe in the future it will repair it, this is the developer's decision
> 
> Can you autists not spam my issue with this shit.

I'm not talking to you, I'm talking to @girpa3
